### ISSUE TITLE
v10.11.0 not found

### ISSUE THREAD

**Issue by jnst** [nodenv/nodenv#109]
```
$ brew update
$ brew upgrade nodenv node-build
```
```
$ nodenv --version
nodenv 1.1.2

$ node-build --version
node-build 3.0.18
```

v10.10.0 is latest in nodenv.

```
$ nodenv install -l
Available versions:

  (...)

  0.11.15
  0.11.16
  0.12.0
  0.12-dev
  0.12-next
  0.12.1
  0.12.2
  0.12.3
  0.12.4
  0.12.5
  0.12.6
  0.12.7
  0.12.8
  0.12.9
  0.12.10
  0.12.11
  0.12.12
  0.12.13
  0.12.14
  0.12.15
  0.12.16
  0.12.17
  0.12.18
  10.0.0
  10.1.0
  10.2.0
  10.2.1
  10.3.0
  10.4.0
  10.4.1
  10.5.0
  10.6.0
  10.7.0
  10.8.0
  10.9.0
  10.10.0            <------------- here
  4.0.0
  4.x-dev
  4.x-next
  4.1.0
  4.1.1
  4.1.2
  4.2.0
  4.2.1
  4.2.2

  (...)

```

**Comment by jasonkarns** [comment]
10.11.0 was added 11 days ago: https://github.com/nodenv/node-build/commit/750674b4c70841b3548d3f1e722c29f227c8d84d

Reminder that a git installation will always have the latest node versions (instead of waiting for the version of node-build to be bumped, and again for homebrew to update its formula) 
https://github.com/nodenv/nodenv/wiki/FAQ#available-node-versions-are-out-of-date

(there's also https://github.com/nodenv/node-build-update-defs to scrape definitions directly from nodejs.org)

**Comment by jasonkarns** [comment]
regardless, [3.0.19](https://github.com/nodenv/node-build/releases/v3.0.19) has now been released including 10.11.0 😸
